<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1163</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="sidebar" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>AI推拉流测试工具</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="typeManagerBtn">
         <property name="text">
          <string>视频类别自定义管理</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="videoManagerBtn">
         <property name="text">
          <string>视频管理</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="typeManager">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>75</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>70</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>类别</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_16" native="true"/>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_4">
             <property name="text">
              <string>backup •</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="TypeSearchBtn">
             <property name="text">
              <string>查询</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="typeResetBtn">
             <property name="text">
              <string>重置</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="addType">
             <property name="text">
              <string>新增类别</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>915</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="typeTableWidget"/>
        </item>
        <item>
         <widget class="QWidget" name="widget_8" native="true">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>70</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QPushButton" name="typePrevBtn">
             <property name="text">
              <string>上一页</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="typeNextBtn">
             <property name="text">
              <string>下一页</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="videoManager">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QWidget" name="widget_7" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_14">
           <item>
            <widget class="QWidget" name="widget_3" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="videoName_text">
                <property name="text">
                 <string>视频名称</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit"/>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_4" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="videoType_text">
                <property name="text">
                 <string>视频类别</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_2"/>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_5" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="streamStatus_text">
                <property name="text">
                 <string>推流状态</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_3"/>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_6" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QPushButton" name="videoSearchBtn">
                <property name="text">
                 <string>查询</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="videoResetBtn">
                <property name="text">
                 <string>重置</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_15" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_15">
           <item>
            <widget class="QPushButton" name="initConfigBtn">
             <property name="text">
              <string>初始化配置</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="videoImport">
             <property name="text">
              <string>视频导入</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="history">
             <property name="text">
              <string>历史推流记录</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="videoTableWidget">
          <column>
           <property name="text">
            <string>类别名称</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>描述</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>创建时间</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>最后修改时间</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>操作人</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>启用状态</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_9" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_16">
           <item>
            <widget class="QWidget" name="widget_13" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_12">
              <item>
               <widget class="QPushButton" name="videoSearchBtn_2">
                <property name="text">
                 <string>上一页</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="videoResetBtn_2">
                <property name="text">
                 <string>下一页</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1163</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
